flow-adapters/build.gradle

```diff
@@ -6,8 +6,21 @@ dependencies {
     testCompile project(':reactive-streams-tck')
     testCompile group: 'org.testng', name: 'testng', version: '5.14.10'
 }
+
+jar {
```

Comment:
This appears twice.

Reply:
@akarnokd Nice catch, I've fixed that now :)